modules/aws-installing-an-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,373 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="aws-installing-an-aws-load-balancer-operator_{context}"]
+= Installing an AWS Load Balancer Operator
+You can install an AWS Load Balancer Operator (ALBO) if you meet certain requirements.
+
+.Prerequisites
+
+* You have an existing
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA)
+endif::openshift-rosa[]
+cluster with bring-your-own-VPC (BYO-VPC) configuration across multiple availability zones (AZ) installed in STS mode.
+* You have access to the cluster as a user with the `dedicated-admin` role.
+* You have access to modify the VPC and subnets of the created
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster.
+* You have installed the
+ifndef::openshift-rosa[]
+{product-title}
```

Comment:
If this gets added to OCP, I think you will have a prereq to install the OCP CLI and oc CLI.

Reply:
This is actually a topic in OCP which has to do with the AWS Load Balancer Operator; however, there are significant differences per the PM hence the new topic.